youths/schema.py

```diff
@@ -0,0 +1,176 @@
+from datetime import datetime
+from django.core import signing
+from django.core.exceptions import ObjectDoesNotExist
+from django_ilmoitin.utils import send_notification
+import graphene
+from graphene_django.types import DjangoObjectType
+from graphql_jwt.decorators import login_required
+
+from profiles.models import Profile
+
+from .consts import GENDERS, LANGUAGES
+from .models import YouthProfile
+from .utils import create_approval_token, mail_encrypted_approval_link
+
+
+class YouthProfileType(DjangoObjectType):
+    class Meta:
+        model = YouthProfile
+        exclude = ["id", "approval_token"]
+
+
+PreferredLanguage = graphene.Enum(
+    "PreferredLanguage", [(l[0].upper(), l[0]) for l in LANGUAGES]
+)
+Gender = graphene.Enum("Gender", [(g[0].upper(), g[0]) for g in GENDERS])
+
+
+# Abstract base fields
+class YouthProfileFields(object):
```

Comment:
python 3 uses `object` as default base for all classes in nothing else is specified, so you can drop it from this line 🙂

Reply:
but maybe you can use `graphene.InputObjectType` as base here, then another input class will just extend this?

Reply:
I originally used `AbstractType` but it's deprecated so I followed the update [guide](https://github.com/graphql-python/graphene/blob/master/UPGRADE-v2.0.md#abstracttype-deprecated). You make an excellent point though, since the point of the AbstractType was that it can be mounted as an `InputObjectType` or a `ObjectType`, but since we use it only as an input, there's no need to inherit from the generic `object`.